leakcanary-analyzer/src/test/java/com/squareup/leakcanary/AsyncTaskLeakTest.java

```diff
@@ -49,6 +49,9 @@
     return asList(new Object[][] {
         { ASYNC_TASK_PRE_M }, //
         { ASYNC_TASK_M }, //
+        // TODO With the latest perflib, all the tests for this O Heap Dump are failing because
+        // we're unable to find a "mBuffer" field in Bitmap when computing the retained size.
+        // Perflib added new code related to bitmaps so there's probably something there.
```

Comment:
huh that existed prior: https://github.com/square/leakcanary/commit/b818140d3f5ed95430360ab07efe39f73e6a845d

Reply:
just ignoring when the field is missing.